src/sentry/apidocs/hooks.py

```diff
@@ -70,10 +71,47 @@ def __get_explicit_endpoints() -> List[Tuple[str, str, str, Any]]:
     ]
 
 
+# def codemod_deprecate_public(endpoints):
```

Comment:
I will remove this comment block. Just adding it here so you see how code changes happened.